src/arrayMethodSort.js

```diff
@@ -1,11 +1,24 @@
 'use strict';
 
-/**
- * Implement method Sort
- */
+const compare = (a, b) => {
+  return a.toString() > b.toString() ? 1 : -1;
+};
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction = compare) {
+    let valueCopy = 0;
```

Comment:
The variable `valueCopy` is declared outside of the loop, which is fine, but it could be declared inside the block where it's used for better encapsulation of its scope.